### DISCUSSION TITLE
希望可以批量某标签里的进展记事转移

### DISCUSSION THREAD

**Discussion by vivzer** [WatsonYao/nian_app#81] [Q&A] [answered]
nian已经用了很多年，标签当初建立的时候有些不完善，想重新整理，但标签里的进展太多，需要一个批量转移的功能，希望能够开发一个

**Comment by WatsonYao** [comment, marked as answer]
标准记本右上角点开，支持“标签批量增删”。

**Reply by yuggbh** [reply]
请问标签替换的功能可以通过此项实现吗？即把多个标签的内容转移到一个标签

**Reply by yuggbh** [reply]
或者说是否有标签融合的功能

**Reply by WatsonYao** [reply]
无此类计划，可以先添加新的后删除旧的，绕一下实现替换的功能。
